### ISSUE TITLE
Handle the case where not all individuals have the same keypoints.

### ISSUE THREAD

**Issue by niksirbi** [neuroinformatics-unit/movement#150]
Currently our data structure assumes that all individuals (animals) are tracked with the same keypoints (body parts). This assumption holds for most, **but not all** pose estimation datasets.

We should consider this case and decide what to do about it:
- figure out a technical workaround for supporting such cases
- or explicitly state that they are not supported

**Comment by sfmig** [comment]
For the time being we should probably document this more clearly. It doesn't seem to be in the Input/output section or in the API reference for the relevant functions, maybe those are nice places to put this?

**Comment by niksirbi** [comment]
Agreed, document for the short term, and think about supporting this in the long-term—in the sense of loading such data into separate `movement` datasets (1 per skeleton type).

**Comment by sfmig** [comment]
The freemocap example (PR #633) demos an easy way to combine all keypoints along a single dimension
